lib/all-badge-examples.js

```diff
@@ -1299,6 +1299,14 @@ const allBadgeExamples = [
           'python'
         ]
       },
+      {
+        title: 'PyPI',
```

Comment:
The title should be **PyPI – Django version** or maybe just **Django version**, and it's probably worth adding `pypi` to keywords.

Reply:
I've named it consistently with all the other PyPI badges:


![screenshot at 2017-11-30 20 03 58](https://user-images.githubusercontent.com/6025893/33452836-65626ec2-d60b-11e7-9d5b-4be723e51284.png)


Shall I update the others to "PyPI - License", "PyPI - Format" and so on as well?

Reply:
Sure! That sounds great.